Source/Core/DolphinQt/Debugger/BreakpointWidget.cpp

```diff
@@ -308,7 +316,11 @@ void BreakpointWidget::OnSave()
 
 void BreakpointWidget::AddBP(u32 addr)
 {
-  PowerPC::breakpoints.Add(addr);
+  AddBP(addr, false, true, true);
```

Comment:
Is there a reason it's logging by default? Is this method still used? If not, it should be removed.

There is also a missing newline after the function. Not sure why the lint bot didn't catch this.

Reply:
I believe "write to log and break" was always the default? The log function does still work; I've been using it to monitor function calls.